benchmarks/htap/run.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+from datetime import datetime
+
```

Comment:
The above 2 modules haven't been used and can be removed.

Reply:
done